release/models/relay-agent/openconfig-relay-agent.yang

```diff
@@ -624,6 +633,62 @@ module openconfig-relay-agent {
 
   }
 
+  grouping relay-agent-ipv4-helper-address-config {
+    description
+      "Configuration options for an IP helper-address";
+
+    leaf address {
+      type oc-inet:host;
+      description
+        "IP address to relay DHCP requests to";
+    }
+
+    leaf local-address {
```

Comment:
In most of the OC models, we use source-address as the leaf describing the origin a device should use to create a packet.  

```suggestion
    leaf source-address {
```